test/e2e/README.md

```diff
@@ -0,0 +1,54 @@
+# End-to-End Tests
+
+This section contains information about end-to-end (e2e) tests for the VPP agent.
```

Comment:
It'd be good to mention the tests are built upon the pkg and pattern from Ligato: `go.ligato.io/vpp-agent/v3/tests/e2e`.  Is there any Ligato test readme/doc to link to here as well?